README.md

```diff
@@ -32,6 +32,22 @@ manage a new cloud-native application. This enables Crossplane users to:
 The full vision and architecture of the Crossplane project is described in our
 [architecture document].
 
+## Infrastructure Stacks
+
+The following infrastructure stacks are currently available for use with Crossplane. If you maintain an infrastructure stack that is not listed below, please open a PR to add it to the list!
+
+Maintained by Crossplane Org:
+
+* [stack-aws]
+* [stack-azure]
+* [stack-rook]
+* [stack-gcp]
+
+Maintained by external communities:
+
+* [stack-packet]
+* [stack-cloudscale]
```

Comment:
Do you think there's value in making a distinction around who maintains which stack in this context? I lean toward not doing so.

Reply:
I think it would be fine not to do so. The benefit to me is pointing out that there are stacks outside of the crossplane org and also denoting the stacks that we are committed to ensuring are kept up-to-date and usable.

Reply:
I'm hoping to keep the README succinct and limited to "need to know" splash page level information so that people can parse it quickly.

I think listing known stacks meets that criteria, but I don't think we need to clarify who owns and maintains them at this level. Hopefully it will be obvious to anyone who clicks through who is maintaining each stack from their Github org and README.